WeakAuras/WeakAuras.lua

```diff
@@ -4127,6 +4127,19 @@ function WeakAuras.Modernize(data)
 
   end
 
+  -- Introduced in July 2020 in Shadolands
+  if data.internalVersion < 34 then
+    if data.regionType == 'dynamicgroup'
+    and (data.grow == "CIRCLE" or data.grow == "COUNTERCIRCLE")
+    and data.fullCircle == nil then
```

Comment:
data.fullCircle will always be true here. 

We unfourtunately validate the settigns of a aura against the defaults before Modernize. (Which looks like a bug to me.) And that Modernize will add the missing fullCircle bool.

I think just removing the data.fullCircle == nil here should work though.

Reply:
Ah, right. I forgot about that.

Reply:
You should test it once against old settings :)